src/components/CardSelect/components/CardChips.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ReactElement } from 'react'
+import { Chips } from '../../Chips'
+import { TChipsProps } from '../../Chips/types'
+
+interface TCardChips {
+  chips: TChipsProps[]
+  disabled?: boolean
+}
+export const CardChips = (props: TCardChips): ReactElement => {
+  const { chips, disabled } = props
+  return (
+    <>
+      {chips.map((chip, index) => (
+        <Chips
+          {...chip}
+          key={index}
```

Comment:
it's better to have a unique ID for every element. If you don't have it add by uuid before mapping